docs/PRODUCTION.md

```diff
@@ -13,3 +13,13 @@ The postgres and helm charts are included as conditionals in the Gen3 [umbrella
   repository: "https://charts.bitnami.com/bitnami"
   condition: global.dev
   ```
+
+### Kubernetes Configmap and Secret Configuration
+For the seamless operation of our services, we utilize Kubernetes secrets. To streamline the integration and management of these secrets, we highly recommend deploying External Secret Manager alongside any existing secret management systems you may already have in place. For a comprehensive guide and best practices on implementing External Secrets within our ecosystem, please consult our dedicated External Secrets Documentation available [here](https://github.com/uc-cdis/gen3-helm/blob/master/docs/external_secrets.md).
+
+Our services also utilize non-secret configuration variables provided via Kubernetes ConfigMaps. For streamlined management, we advise keeping your values.yaml and configuration files files in source control and utilizing ArgoCD for automatic updates and efficient management of your Gen3 Helm chart.
+
+Each service is designed to seamlessly integrate and manage the combination of Kubernetes secrets and ConfigMaps, ensuring the encapsulated information is effectively injected into the underlying application.
+
+Please see the diagram provided [here](https://lucid.app/lucidchart/f0e9baf8-9179-4be0-ae65-182ed891df22/edit?invitationId=inv_83c83c6a-0f1d-4236-a13e-3131c1fc851f&page=TDBgA8eMNr7J#) that details how External Secrets operates. We also mention the use of Argo CD as our choice option for Helm deployments.
```

Comment:
I think we need to actually embed the image. The community is not going to have access to our Lucid and I'm not sure about making a public diagram. I just meant we should keep the source link so that if we need to regenerate the image, it's there